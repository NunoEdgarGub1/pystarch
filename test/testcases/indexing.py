x01 = [1,2,3,4,5]
x02 = x01[3]
x03 = (0, 'bob', {'a': 1})
x04 = x03[0]
x05 = x03[1]
x06 = x03[2]
x07 = x03[2]['a']
