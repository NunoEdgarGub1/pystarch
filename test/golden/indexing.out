x01 List(Num) [1, 2, 3, 4, 5]
x02 Num
x03 Tuple(Num,Str,Dict(Str,Num)) (0, 'bob', {'a': 1})
x04 Num
x05 Str
x06 Dict(Str,Num)
x07 Num
